crates/sui-cluster-test/Cargo.toml

```diff
@@ -16,7 +16,6 @@ jsonrpsee = { version = "0.15.1", features = ["full"] }
 tracing = { version = "0.1.36", features = ["log"] }
 clap = { version = "3.1.14", features = ["derive"] }
 reqwest = { version = "0.11.11", features = ["blocking", "json"] }
-telemetry-subscribers.workspace = true
```

Comment:
Can we keep the workspace dependency setup? It will make it easier to put narwhal back into a separate repo.

Reply:
There are zero plans for putting narwhal back into a separate repo. This is not going to happen any time within the next 1-2 years